hack/bundle-automation/charts-config.yaml

```diff
@@ -1,12 +1,22 @@
 
 - repo_name: "managed-serviceaccount"
   github_ref: "https://github.com/stolostron/managed-serviceaccount.git"
-  charts:
+  branch: "backplane-2.4"
+  # charts:
+  #   - name: "managed-serviceaccount"
+  #     chart-path: "charts/managed-serviceaccount"
+  #     always-or-toggle: "toggle"
+  #     imageMappings:
+  #       managed-serviceaccount: managed_serviceaccount
+  #     inclusions:
+  #       - "pullSecretOverride"
+  template_addons:
     - name: "managed-serviceaccount"
-      chart-path: "charts/managed-serviceaccount"
+      yaml-paths:
```

Comment:
actually why we use yaml but not helm chart?

Reply:
the manifest files we used to deploy the agent upstream are yaml-format, which does not support helm format.